doc/developers/develop.rst

```diff
@@ -228,8 +228,8 @@ to slice rows and columns.
 
 .. deprecated:: 0.24
 
-    The _pairwise attribute is deprecated in 0.24. From 0.26 onward,
-    the `pairwise` estimator tag should be used instead.
+    The _pairwise attribute is deprecated in 0.24. From 1.1 onward
+    (renaming of 0.26), the `pairwise` estimator tag should be used instead.
```

Comment:
```suggestion
    The _pairwise attribute is deprecated in 0.24. From 1.1 (renaming of 0.26)
    onward, the `pairwise` estimator tag should be used instead.
```

Reply:
I a bit confused, is it the opposit of what Nicolas wanted or that you proposed the same change?

Reply:
It's NBD, merging ;)